examples/complete_workflow/AuthoritationCodePKCECompleteWorkFlow.java

```diff
@@ -0,0 +1,135 @@
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeRefreshRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeUriRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.pkce.AuthorizationCodePKCERefreshRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.pkce.AuthorizationCodePKCERequest;
+import com.wrapper.spotify.requests.data.player.GetCurrentUsersRecentlyPlayedTracksRequest;
+
+public class AuthoritationCodePKCECompleteWorkFlow {
+	
+	/*
+	 *This method is the most appropriate for desktop applications, as we will not use the clientSecret of our app, only the clientId, which we will get from Spotify Developer DashBoard 
+	 *after registering our app. Instead we will use two codes, the codeVerifier and the codeChallenge.  Instead, your application should generate a code verifier and a code challenge 
+	 *before each authentication request.The code verifier is a cryptographically random string between 43 and 128 characters in length. It can contain letters, digits, underscores, 
+	 *periods, hyphens, or tildes. To generate the code challenge, your app should hash the code verifier using the SHA256 algorithm. Then, base64url encode the hash that you generated. 
+	 *We will also need the redirection URI, to which the user will be redirected after authorising his account to obtain the code that is exchanged for the access and refresh tokens.
+	 *This URI must be registered in our app in Spotify Developer DashBoards.
+	 * 
+	 */
+
+	  private static final String clientId = "zyuxhfo1c51b5hxjk09x2uhv5n0svgd6g";
+	  private static final URI redirectUri = SpotifyHttpManager.makeUri("https://example.com/spotify-redirect");
+	  private static final String codeChallenge = "w6iZIj99vHGtEx_NVl9u3sthTN646vvkiP8OMCGfPmo";
+	  private static final String codeVerifier = "NlJx4kD4opk4HY7zBM6WfUHxX7HoF8A2TUhOIPGA74w";
+	  
+	  /*
+	   * Add the properties to our spotifyApi object
+	   */
+	  private static final SpotifyApi spotifyApi = new SpotifyApi.Builder()
+	    .setClientId(clientId)
+	    .setRedirectUri(redirectUri)
+	    .build();
+
+	  /**
+	   * <p>We make the request to obtain the authorisation URI that will allow us, after the redirection, to obtain the code we need to 
+	   * obtain the access and refresh token. The URI shown must be copied in our browser. The scope field indicates what we will be able 
+	   * to do with the user's information. This will be shown to them before they authorise our app.</p>
+	   */
+	  public static void obtainsCode() {
+		 final AuthorizationCodeUriRequest authorizationCodeUriRequest = spotifyApi.authorizationCodePKCEUri(codeChallenge)
+			        .scope("user-read-recently-played")
+				    .build();
+	    final URI uri = authorizationCodeUriRequest.execute();
+	    System.out.println("The URI you have to use in the browser is the following: " + uri.toString());
```

Comment:
Please add wording that more clearly indicates what's meant by "use in the browser".